distribution/lib/Standard/AWS/0.0.0-dev/src/Database/Redshift/Internal/Redshift_Dialect.enso

```diff
@@ -67,17 +67,10 @@ type Redshift_Dialect
         Base_Generator.truncate_table_delete_from_style self table_name
 
     ## PRIVATE
-       Generates SQL for limiting the number of rows using the `LIMIT` clause.
-    get_limit_part_sql : Integer -> SQL_Builder
-    get_limit_part_sql self limit =
-        SQL_Builder.code (" LIMIT " + limit.to_text)
-
-    ## PRIVATE
-       Generates SQL for limiting the number of rows using the `TOP` clause.
-    get_top_part_sql : Integer -> SQL_Builder
-    get_top_part_sql self limit =
-        _ = [limit]
-        SQL_Builder.code ""
+       Generates SQL modifier for limiting the number of rows and its position in the query
+    get_limit_sql_modifier : Integer -> Any
```

Comment:
I'd prefer if we could make the type more precise. Could this be just `Context_Extension`?

Reply:
But I guess the pair is OK too, it would just be good to document somewhere what should be put there so that future implementors do not have to rely solely on examples and usage,

Reply:
I'd like to revisit the approach at some point. I am unsure about the "magic numbers" for the position in the SQL.